frontend/helper/GPSHelper.js

```diff
@@ -0,0 +1,27 @@
+import { Platform } from 'react-native'
+import { Permissions, Location } from 'expo'
+
+import { store } from '../App'
+import { updateGPS } from './GPSHelper.Action';
+
+export const getLocationAsync = async () => {
```

Comment:
if there is not a function `getLocation` that do not work asynchronous, i would prefer not to write the `-Async` postfix to the variable name.